packages/ui/src/components/ChatMessage/ChatContent.native.tsx

```diff
@@ -0,0 +1,351 @@
+import { utils } from '@tloncorp/shared';
+import {
+  Block,
+  Story,
+  VerseBlock,
+  VerseInline,
+  isImage,
+} from '@tloncorp/shared/dist/urbit/channel';
+import {
+  Inline,
+  isBlockCode,
+  isBlockquote,
+  isBold,
+  isBreak,
+  isInlineCode,
+  isItalics,
+  isLink,
+  isShip,
+  isStrikethrough,
+} from '@tloncorp/shared/dist/urbit/content';
+import { AVPlaybackStatus, ResizeMode, Video } from 'expo-av';
+import { ReactElement, useEffect, useRef, useState } from 'react';
+import { Linking } from 'react-native';
+
+import { Image, Text, View, XStack, YStack } from '../../core';
+import { Button } from '../Button';
+import ContactName from '../ContactName';
+
+function ShipMention({ ship }: { ship: string }) {
```

Comment:
Most of the components here don't need to be native-specific -- can we split it so that the minimum possible is in the native file? For example if `Video` component from `expo-av` is native-only, I'd have a `Video.native.ts`, but still have `BlockContent` with wrapper styles, etc. be shared.